src/scss/page/auth.scss

```diff
@@ -121,6 +121,9 @@ html.bodyAuthDeleted {
 	.frame { width: 830px; }
 	.title { font-family: 'Riccione'; font-size: 102px; line-height: 90px; letter-spacing: -3px; font-weight: 500; margin-bottom: 24px; color: $colorPrimary; }
 	.label { font-size: 16px; line-height: 26px; letter-spacing: -0.28px; width: 448px; margin: 0px auto 22px auto; color: $colorSecondary; }
+	.footer {
+		.disclaimer { color: $colorTertiary; }
```

Comment:
You used .label.disclamer in footer.scss, here is just .disclamer - make consistent pls.